src/schema/provenance_schema.yaml

```diff
@@ -659,6 +659,16 @@ ENTITIES:
           - validate_in_admin_group
         before_property_update_validators:
           - validate_in_admin_group
+      priority_project_list:
```

Comment:
Because Publication inherits Dataset properties, we'll need to explicitly specify this new field and mark it with `null` to exclude from Publication.

Reply:
Yep, good catch. Fixed